protocol-units/settlement/mcr/setup/src/deploy/mod.rs

```diff
@@ -144,39 +125,40 @@ impl Deploy {
 			.and_then(|transactions| transactions.get(5))
 			.and_then(|transaction| transaction.as_object())
 			.and_then(|transaction_object| transaction_object.get("contractAddress"))
-			.ok_or(anyhow!(
-				"No contract address in forge script exec deployment result file."
-			))
+			.ok_or(anyhow!("No contract address in forge script exec deployment result file."))
 			.map(|v| {
 				let s = v.as_str().expect("Contract address elements should be strings");
 				s.to_owned()
 			})?;
 
 		// generate random well-known accounts and addresses
-		let mut well_known_account_private_keys = if let Some(existing_testing_config) = config.testing.clone() {
-			existing_testing_config.well_known_account_private_keys
-		} else {
-			let mut keys = Vec::new();
-			for _ in 0..10 {
-				let wallet = PrivateKeySigner::random();
-				keys.push(wallet.to_bytes().to_string());
-			}
-			keys
-		};
+		// let mut well_known_account_private_keys =
+		// 	if let Some(existing_testing_config) = config.testing.clone() {
+		// 		existing_testing_config.well_known_account_private_keys
+		// 	} else {
+		// 		let mut keys = Vec::new();
+		// 		for _ in 0..10 {
+		// 			let wallet = PrivateKeySigner::random();
+		// 			keys.push(wallet.to_bytes().to_string());
+		// 		}
+		// 		keys
+		// 	};
```

Comment:
Is this meant to be switched off temporarily?

Reply:
I should be removed. I keep it because it's @l-monninger code and I change the behavior. I would like to discuss with him if it's ok.